web/src/gamesShared/components/fbg/GameDarkSublayout.tsx

```diff
@@ -74,7 +74,13 @@ export class GameDarkSublayoutInternal extends React.Component<IGameDarkSublayou
         <Typography
           variant="h6"
           gutterBottom={true}
-          style={{ float: 'left', marginTop: '10px', backgroundColor: 'red', color: 'white' }}
+          style={{
+            float: 'left',
+            marginTop: '10px',
+            backgroundColor: 'red',
+            color: 'white',
+            whiteSpace: 'nowrap',
```

Comment:
For clarification: I added the `whiteSpace`property to prevent the game name to be wrapped around. Without this fix, the CSS flex logic would occasionally decide to wrap "Estate Buyer" after "Estate" so that "Buyer" is outside of the 49 pixel height of the top bar.

Reply:
SGTM, thanks!